tests/app/shared/issue-tables/issue-sorter.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { getSortedData } from '../../../../src/app/shared/issue-tables/issue-sorter';
+import { MatSort } from '@angular/material';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION, ISSUE_PENDING_MODERATION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+
+describe('issuer-sorter', () => {
+  describe('getSortedData()', () => {
+    const dummyTeam = new Team({
+      id: 'F09-2',
+      teamMembers: [],
+    });
+    const dummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherDummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
+    const issuesList: Issue[] = [dummyIssue, otherDummyIssue];
+
+    const moderationIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherModerationIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION, dummyTeam);
+    const todoIssuesList: Issue[] = [moderationIssue, otherModerationIssue];
+    const matSort: MatSort = new MatSort();
+
+    it('sorts issues based on their assignees correctly', () => {
+      matSort.active = 'assignees';
+      matSort.direction = 'asc';
+      const sortedIssuesByAssigneesAsc = getSortedData(matSort, issuesList);
+      assertOrder(sortedIssuesByAssigneesAsc, dummyIssue, otherDummyIssue);
+
+      matSort.direction = 'desc';
+      const sortedIssuesByAssigneesDesc = getSortedData(matSort, issuesList);
+      assertOrder(sortedIssuesByAssigneesDesc, otherDummyIssue, dummyIssue);
+    });
+
+    it('sorts issues based on their string fields correctly', () => {
+        matSort.active = 'title';
+        matSort.direction = 'asc';
+        const sortedIssuesByTitleAsc = getSortedData(matSort, issuesList);
+        assertOrder(sortedIssuesByTitleAsc, dummyIssue, otherDummyIssue);
+
+        matSort.direction = 'desc';
+        const sortedIssuesByTitleDesc = getSortedData(matSort, issuesList);
+        assertOrder(sortedIssuesByTitleDesc, otherDummyIssue, dummyIssue);
+    });
+
+    it('sorts issues based on their integer fields correctly', () => {
+        matSort.active = 'id';
+        matSort.direction = 'asc';
+        const sortedIssuedByIdAsc = getSortedData(matSort, issuesList);
+        assertOrder(sortedIssuedByIdAsc, otherDummyIssue, dummyIssue);
+
+        matSort.direction = 'desc';
+        const sortedIssuedByIdDesc = getSortedData(matSort, issuesList);
+        assertOrder(sortedIssuedByIdDesc, dummyIssue, otherDummyIssue);
+    });
+
+    it('sorts issues based on their todos left correctly', () => {
+      matSort.active = 'Todo Remaining';
+      matSort.direction = 'asc';
+      const sortedIssuesByTodoAsc = getSortedData(matSort, todoIssuesList);
```

Comment:
Looks like TSLint doesn't catch the indentation problems in this file.
Based on TSLint's issue tracker, the indentation check seems to be faulty.
Since TSLint became deprecated in 2019, we can consider using Prettier to format indentation, or we can switch to [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint). (low priority for now, since most of the other TSLint checks work)

Reply:
Hmm... that's weird. I will set my indentation to 2 spaces and push these changes before merging.

Reply:
It would be good if we are able to raise this as an issue and work on it in the future!